Airports/EGCN/Airspace_Bases.txt

```diff
@@ -22,4 +22,6 @@
 ;CTA-11
 FL60:N053.22.53.382:W001.16.41.801
 ;CTA-12
-FL60:N053.20.14.119:W001.13.31.214
\ No newline at end of file
+FL60:N053.20.14.119:W001.13.31.214
+;CTA-13
+FL85:N053.18.09.581:W000.49.43.565
```

Comment:
Could you change this to FL85*E to indicate this is Class E? (Thanks!)

Reply:
Done :)